aspnet/client-side/yeoman.rst

```diff
@@ -30,37 +30,47 @@ Install Node.js, npm, and Yeoman
 
 -  The ``–g`` flag installs the generator globally, so that it can be used from any path.
 
-Create an ASP.NET app
--------------------------
+Create an ASP.NET App
```

Comment:
@BrennanConroy I got similar warning (on Windows 10) when I didn't have the final RC2 bits. The warnings might go away with final RC2 and we can reshoot the image.